src/Deserializers/EntityDeserializer.php

```diff
@@ -59,6 +59,11 @@ public function deserialize( $serialization ) {
 	 */
 	protected abstract function getPartiallyDeserialized( array $serialization );
 
+	/**
+	 * @param array $serialization
+	 *
+	 * @return Entity
```

Comment:
Sure. So will you change it? Or let me change it? If not, that's fine, though then I'm sticking with my +1.